include/gatekeeper_lls.h

```diff
@@ -19,6 +19,183 @@
 #ifndef _GATEKEEPER_LLS_H_
 #define _GATEKEEPER_LLS_H_
 
-int run_lls(void);
+#include <rte_timer.h>
+
+#include "gatekeeper_mailbox.h"
+#include "gatekeeper_net.h"
+
+/*
+ * Maximum key length (in bytes) for an LLS map. It should be set
+ * to the maximum key length needed. Currently, this is set to 16 for
+ * the number of bytes in an IPv6 address.
+ */
+#define LLS_MAX_KEY_LEN (16)
+
+/* Number of records that a LLS cache can hold. */
+#define LLS_CACHE_RECORDS (1024)
+
+/* Requests that can be made to a cache. */
+enum lls_req_ty {
+	/* Do a one-time resolution; do not receive further updates. */
+	LLS_REQ_RESOLVE,
+	/* Request to receive a resolution and any updates for this record. */
+	LLS_REQ_SUBSCRIBE,
+	/* Remove a previously-submitted hold. */
+	LLS_REQ_PUT,
+	/* Modify a record, possibly with a new address or port. */
+	LLS_REQ_MODIFY,
```

Comment:
I was thinking that since other blocks (the dynamic configuration) will eventually be able to change an entry, that the interface for doing so should be created here. But I see your point: the LLS can change entries without needing to go through the mailbox.